### ISSUE TITLE
Option to disable grabbing colours of coverart when playing music

### ISSUE THREAD

**Issue by amisukarei** [Malopieds/InnerTune#48]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

When nothing is playing the app uses the monet colours given by the system. When music is playing the UI colour changes and grabs the colour from the cover art of the music that's now playing.
It would be nice if we had an option to disable that and just always have monet colours.

### Why do you want this feature?

I like the material you colouring of the app.

### Additional information

![Screenshot_20240406_104443_InnerTune](https://github.com/Malopieds/InnerTune/assets/83230845/7bf61c90-6e82-4179-8b2e-14db19a57784)
When not playing music

![Screenshot_20240406_104501_InnerTune](https://github.com/Malopieds/InnerTune/assets/83230845/8b30802b-180f-4d56-9e07-32393aa90e9e)
When playing music

**Comment by Malopieds** [comment]
I quickly looked into it and it seems harder than it looks. I'll see if it's simple enough to implement.

**Comment by Malopieds** [comment]
It's already implemented. You must have Android 12+ to have it. It's based on your wallpaper.

**Comment by amisukarei** [comment]
> It's already implemented. You must have Android 12+ to have it. It's based on your wallpaper.

You didn't understand what I meant probably. I have android 14. I'm asking that instead of grabbing coverart colours when playing music it always even when playing music it uses money colouring. Currently when playing music it grabs the coverart colours and only when no music is playing it uses money colour theming

**Comment by Malopieds** [comment]
So you want an option to always have monet theming even if a song is played?

**Comment by amisukarei** [comment]
> So you want an option to always have monet theming even if a song is played?

Yes, everywhere except in the fullscreen music player (see screenshot if you don't know what I mean)

![Screenshot_20240406_190613_InnerTune.jpg](https://github.com/Malopieds/InnerTune/assets/83230845/464a9cf4-6bff-4624-bccf-c29bbfac26c9)

**Comment by Malopieds** [comment]
I'll see what I can do about it. But I think just for the full screen player is a lot for a small change. Doing every part of the app follow monet is fairly easy.
